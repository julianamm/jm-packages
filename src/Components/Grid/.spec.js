import React from 'react'
import { render } from '../../../test.setup'
import renderer from 'react-test-renderer'

import Container from '.'

describe('Container', () => {
    it('Rendering component without crashing', () => {
        const tree = renderer.create(<Container>Text 2</Container>).toJSON()
        expect(tree).toMatchSnapshot()
    })
})