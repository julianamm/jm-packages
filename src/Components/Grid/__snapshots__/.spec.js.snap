// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Rendering Container without crashing 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 100%;
  margin: 0 auto;
  padding: 0;
}

<div
  className="c0"
>
  Text 2
</div>
`;
